src/core/dbus-execute.c

```diff
@@ -2470,7 +2470,7 @@ int bus_exec_context_set_transient_property(
                                                                          -1,
                                                                          c->syscall_filter,
                                                                          SECCOMP_PARSE_PERMISSIVE |
-                                                                         SECCOMP_PARSE_ALLOW_LIST | invert_flag,
```

Comment:
are you sure it's always 0?

Reply:
Yes. This block is called only when the syscall filter was empty and requested filter is an allow-list. Hence `invert_flag == 0`.